crates/trigger/src/cli/stdio.rs

```diff
@@ -95,6 +103,39 @@ impl<F: RuntimeFactors, U> ExecutorHooks<F, U> for StdioLoggingExecutorHooks {
         configured_app: &spin_factors::ConfiguredApp<F>,
     ) -> anyhow::Result<()> {
         self.validate_follows(configured_app.app())?;
+
+        if self.refresh_log {
```

Comment:
This is a very long block which now dominates the `configure_app` function.  Consider breaking out e.g. `if self.refresh_log { self.truncate_log_files(); }`